tests/_core/test_spatial_query.py

```diff
@@ -348,3 +352,33 @@ def test_bounding_box_spatial_data(full_sdata):
         d = get_transformation(element, get_all=True)
         new_d = {k.replace("global", "cropped"): v for k, v in d.items()}
         set_transformation(element, new_d, set_all=True)
+
+
+def test_bounding_box_filter_table():
+    ##
```

Comment:
```suggestion
```

Reply:
I think we need to add a pre-commit check just for @LucaMarconato  😉

Reply:
I though the same 🙈